README.md

```diff
@@ -561,6 +561,59 @@ Use it to type in your key and install.
 
 Go back to Step 4A-4 and then finish with Step 4A-5.
 
+---
+## Forks
+
+### Which Fork Should I Use?
+
+If you have to ask, you're not ready for forks.
+
+Begin your research in Discord #custom-forks
+
+Please do not ask about forks outside of that channel.
+
+If you're new, please start with `commaai/master-ci` and use it for two weeks. This is the latest official version with only lateral support. This will give you a good baseline to compare the other forks to.
+
+> [!CAUTION]
+> Some forks will brick your C3X. \
+> Most forks are not made for C3X because C3X is less than a year old.
+> 
+> Some forks contain banned code. \
+> Using it will get you banned by the comma.ai company.
+> 
+> Some forks have nudgeless-lane-change. \
+> Simply clicking the turn signal will move your car to the next lane. \
+> Without any checks. \
+> Yes, it will drive into the car next to you.
+> 
+> Some forks play a blood-curdling goat scream at max volume randomly.
+> 
+> The danger in using forks is not theoretical because it has full control over your car.
+> 
+> There's a real and present danger in using forks, even on the most popular forks.
+
+If you acknowledge the warning above and are still looking to try a fork that supports TSK the following are available to install at your own risk. Please keep in mind that these are community maintained and may not stay up to date:
+
+#### Forks with Long
+
+* `alexandresato/personal3`
+  * A.K.A SatoPilot, the first fork to support TSK long!
+  * Very quick stop-and-go response
+  * Also `alexandresato/extract_secoc_key_btn` includes a TSK key extract button and is rebased with personal3 often.
+* `chrispypatt/sunnypilot`
+  * Chrispypatt's fork of SunnyPilot's in-progress rewrite. 
+  * Maintained as best effort. This may not always be up to date with latest SunnyPilot. Reach out to @Chrispypatt for requests to rebase his fork with SunnyPilot's rewrite.
+  * Includes a TSK keyboard with key caching
+  * Model switcher to easily switch between various models
+
+#### Forks with Lateral Only
```

Comment:
Does sunnypilot's `master-new` support lateral only as a byproduct of continuously syncing upstream?

Reply:
On that note, what about MADS for SP? Or AOL for FP?  How do those interact with the various forks?

Reply:
Yes on the first one.

Not sure if I follow the second question. MADS and AOL work on cars that the forks support. At this time FP doesn't support TSK and thus AOL is also not supported. MADS works.

Reply:
Good idea. I wonder what SatoPilot calls AOL/MADS in his fork

Reply:
SatoPilot calls it MADS as well